libde265.yaml

```diff
@@ -0,0 +1,62 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libde265/APKBUILD
+package:
+    name: libde265
+    version: 1.0.11
+    epoch: 0
+    description: Open h.265 video codec implementation
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: LGPL-3.0-or-later
+environment:
+    contents:
+        packages:
+            - busybox
+            - ca-certificates-bundle
+            - build-base
+            - autoconf
+            - automake
+            - libtool
+            - pkgconf-dev
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: 2f8f12cabbdb15e53532b7c1eb964d4e15d444db1be802505e6ac97a25035bab
+        uri: https://github.com/strukturag/libde265/releases/download/v${{package.version}}/libde265-${{package.version}}.tar.gz
+    - runs: |
+        autoreconf -fvi
+        # This is en/decoder, so performance matters more than size.
+        export CPPFLAGS="${CPPFLAGS/-Os/-O2}"
+        export CXXFLAGS="${CXXFLAGS/-Os/-O2}"
+
+        ./configure \
+          --build="$CBUILD" \
+          --host="$CHOST" \
+          --prefix=/usr \
+          --mandir=/usr/share/man \
+          --disable-sherlock265
+        make
+        make DESTDIR="${{targets.destdir}}" install
+
+        # The upstream do the same in their Ubuntu/PPA packages.
+        mv "${{targets.destdir}}"/usr/bin/dec265 "${{targets.destdir}}"/usr/bin/libde265-dec265
+
+    - uses: autoconf/make
+    - uses: autoconf/make-install
+    - uses: strip
+subpackages:
+    - name: libde265-dev
+      pipeline:
+        - uses: split/dev
+      dependencies:
+        runtime:
+            - libde265
+      description: libde265 dev
+    - name: libde265-examples
+      pipeline:
+        - runs: |
+            mkdir -p "${{targets.subpkgdir}}"/usr/bin/
+            mv "${{targets.destdir}}"/usr/bin/ ${{targets.subpkgdir}}/usr/bin
```

Comment:
this looks a little strange.. wont this take /usr/bin/libde265-dec265 ?

Reply:
+1, wdyt @dlorenc